GDAXSharp/WebSocket/Models/Request/TickerChannel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace GDAXSharp.WebSocket.Models.Request
+{
+    public class TickerChannel
+    {
+        [JsonProperty("type")]
+        public string Type { get; set; }
+
+        [JsonProperty("product_ids")]
+        public List<string> ProductIds { get; set; }
+
+        [JsonProperty("channels")]
+        public List<Channel> Channels { get; set; }
+
+        [JsonProperty("signature")]
+        public string Signature { get; set; }
+
+        [JsonProperty("key")]
+        public string Key { get; set; }
+
+        [JsonProperty("passphrase")]
+        public string Passphrase { get; set; }
+
+        [JsonProperty("timestamp")]
+        public string Timestamp { get; set; }
```

Comment:
should be a `DateTime`

Reply:
I think not, it should be the same as this:
https://github.com/dougdellolio/gdax-csharp/blob/cc835389bc7bd531b3a833f1d398ee9616303fbd/GDAXSharp/Network/HttpRequest/HttpRequestMessageService.cs#L77 Perhaps I'm missing something?

Reply:
oh yeah this needs to be a string